src/test/java/wedlog/address/model/ModelManagerTest.java

```diff
@@ -401,19 +403,19 @@ public void equals() {
 
         // different guest filteredList -> returns false
         String[] guestKeywords = GEORGE.getName().fullName.split("\\s+");
-        modelManager.updateFilteredGuestList(new NameContainsKeywordsPredicate(Arrays.asList(guestKeywords)));
+        modelManager.updateFilteredGuestList((guest) -> false);
```

Comment:
Could you explain the rationale for this change?

Reply:
the NameContainsKeywordsPredicate returns type Predicate<Person> while the method updateFilteredGuestList only takes in arguments of type Predicate<Guest>.

This is just a temporary fix as the focus for my commit was a working implementation of filter feature and i had in mind that this test case for ModelManager was not exactly relevant anymore after the change in implementation. (we should be filtering from respective predicates created)